.github/workflows/sanity-test.yaml

```diff
@@ -364,4 +364,25 @@ jobs:
           path: src/testlog/*
           if-no-files-found: error
           retention-days: 14
-      
\ No newline at end of file
+      
+      - name: Send Github Action failure to Slack
```

Comment:
Can we try a manual run for this to validate the integration?

Reply:
We need to add slack-webhook in the secrets and then can test it. I will let you know once its added.

Reply:
Could we add a similar action to the cleanup scripts?  Would be nice to know when those fail, too.  Thanks!

Reply:
@neha-Gupta1 - created SLACK_WEBHOOK_URL secret

Reply:
As Vaibhav mentioned the secret is added and I have done some testing too. Its ready for review now.